doc/help-scoring.md

```diff
@@ -26,6 +26,11 @@ package page.
 To view packages you have liked, use the **My pub.dev > My liked packages** menu
 option.
 
+## Download counts
+
```

Comment:
Suggested rewrite:

Pub.dev tracks and stores how many times each version of each package gets downloaded. This is used to display package usage metrics.

Reply:
Done